client-hc/src/test/java/com/graphhopper/api/GraphHopperWebIT.java

```diff
@@ -248,6 +248,8 @@ public void doNotReadFinishInstruction() {
 
     @Test
     public void testSimpleExport() {
+        // todonow: is it ok/meant to be that export() always uses get (regardless of the setting), looks like our
+        // POST endpoint does not support gpx yet?
```

Comment:
I guess this is so because the POST endpoint does not support gpx and we can leave as is here?

Reply:
Ah, good catch. I think we should always prefer POST as the request is optionally compressed and has no other limitations (?)

> looks like our POST endpoint does not support gpx yet?

doPost should support it or why do you think it does not?

Reply:
> doPost should support it or why do you think it does not?

Hm, yes you are right it should already support it. Ok then we should make `GraphHopperWeb#export` respect the `postRequest` flag. If I do this the test fails, but maybe thats for another issue.